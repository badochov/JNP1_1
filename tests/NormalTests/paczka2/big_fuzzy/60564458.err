Error in line 4: lDnc S666 78,9
Error in line 9: aMEpYx A34 69,7
Error in line 7: qFOj5Iw S666 73,3
Error in line 11: qFOj5Iw A264 28,4
Error in line 8: lDnc A264 87,4
Error in line 23: aMEpYx S666 78,3
Error in line 17: lDnc S666 77,0
Error in line 14: m1RSQJ1hX A34 83,6
Error in line 43: m1RSQJ1hX S666 46,8
Error in line 51: qFOj5Iw A386 34,6
Error in line 54: lDnc A264 46,4
Error in line 59: 5JiZIK A34 20,4
Error in line 58: qFOj5Iw A264 32,0
Error in line 61: m1RSQJ1hX A34 167,8
Error in line 70: s4r A386 30,9
Error in line 78: aMEpYx A34 97,4
Error in line 85: qFOj5Iw A34 20,0
Error in line 95: s4r A264 36,9
Error in line 100: qFOj5Iw A386 113,8
Error in line 108: 5JiZIK A386 75,2
Error in line 122: qFOj5Iw A386 40,2
Error in line 126: A264 A34 39,2
Error in line 134: 5JiZIK A34 80,3
Error in line 132: qFOj5Iw A34 29,2
Error in line 138: s4r A386 73,4
Error in line 143: 5JiZIK S666 40,89
Error in line 145: 5JiZIK S666 24,1
Error in line 139: aMEpYx A264 95,6
Error in line 148: qFOj5Iw A264 79,1
Error in line 157: s4r A386 97,2
Error in line 171: A264 A34 10,8
Error in line 177: 5JiZIK A386 54,5
Error in line 189: 5JiZIK A34 21,5
Error in line 194: lDnc S666 17,7
Error in line 200: s4r A34 92,4
Error in line 214: aMEpYx A264 &3,8
Error in line 221: aMEpYx A264 17,7
Error in line 226: lDnc S666 97,48
Error in line 232: 5JiZIK A34 
Error in line 233: 5,8
Error in line 230: 5JiZIK A34 18,9
Error in line 227: A264 A264 24,2
Error in line 234: 5JiZIK S666 39,3
Error in line 228: lDnc S666 411,7
