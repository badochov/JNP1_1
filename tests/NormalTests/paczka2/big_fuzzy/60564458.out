aMEpYx A 14,3
m1RSQJ1hX S 84,5
A264 14,3
S666 84,5
S666 84,5
A264 14,3
5JiZIK A 350,4
A264 A 84,7 S 320,4
aMEpYx A 191,3
lDnc A 121,6 S 29,8
m1RSQJ1hX A 36,7 S 317,5
qFOj5Iw A 77,2
A34 98,4
A264 266,5
A386 497,0
S666 667,7
A264 A 84,7 S 320,4
A264 266,5
5JiZIK A 350,4
A264 A 84,7 S 320,4
aMEpYx A 217,1
lDnc A 121,6 S 39,8
m1RSQJ1hX A 45,7 S 317,5
qFOj5Iw A 77,2
s4r A 36,9
A34 124,2
A264 312,4
A386 497,0
S666 677,7
aMEpYx A 217,1
A264 A 84,7 S 320,4
A264 312,4
S666 944,4
aMEpYx A 217,1
5JiZIK A 359,7
A264 A 84,7 S 320,4
aMEpYx A 217,1 S 260,5
lDnc A 121,6 S 161,8
m1RSQJ1hX A 187,5 S 317,5
qFOj5Iw A 86,4 S 266,7
s4r A 133,8
A34 221,1
A264 312,4
A386 657,3
S666 1326,9
5JiZIK A 421,2
A264 A 84,7 S 320,4
aMEpYx A 217,1 S 260,5
lDnc A 121,6 S 161,8
m1RSQJ1hX A 187,5 S 317,5
qFOj5Iw A 86,4 S 266,7
s4r A 133,8
A34 221,1
A264 373,9
A386 657,3
S666 1326,9
5JiZIK A 421,2 S 33,5
A264 A 116,0 S 329,5
aMEpYx A 232,5 S 260,5
lDnc A 175,6 S 169,9
m1RSQJ1hX A 453,3 S 317,5
qFOj5Iw A 232,2 S 316,9
s4r A 321,6
A34 374,6
A264 709,1
A386 868,7
S666 1427,8
5JiZIK A 421,2 S 33,5
A264 A 116,0 S 329,5
aMEpYx A 232,5 S 260,5
lDnc A 175,6 S 169,9
m1RSQJ1hX A 453,3 S 317,5
qFOj5Iw A 232,2 S 316,9
s4r A 321,6
A34 374,6
A264 709,1
A386 868,7
S666 1427,8
5JiZIK A 427,6 S 50,4
A386 1332,1
S666 2013,8
A264 A 561,1 S 529,4
A264 843,2
lDnc A 341,4 S 191,1
aMEpYx A 273,3 S 340,8
5JiZIK A 573,4 S 78,6
A264 A 960,6 S 529,4
aMEpYx A 273,3 S 346,4
lDnc A 2109,1 S 191,1
m1RSQJ1hX A 614,5 S 378,2
qFOj5Iw A 464,4 S 509,4
s4r A 327,9 S 358,1
A34 640,3
A264 3178,6
A386 1504,3
S666 2391,2
S666 2391,2
5JiZIK A 785,9 S 78,6
A264 A 990,1 S 529,4
aMEpYx A 273,3 S 346,4
lDnc A 2109,1 S 191,1
m1RSQJ1hX A 614,5 S 378,2
qFOj5Iw A 464,4 S 509,4
s4r A 327,9 S 358,1
A34 640,3
A264 3178,6
A386 1746,3
S666 2391,2
